test/development/basic/barrel-optimization.test.ts

```diff
@@ -27,7 +28,9 @@ describe('optimizePackageImports', () => {
         scripts: {
           setup: `cp -r ./node_modules_bak/* ./node_modules`,
           build: `yarn setup && next build`,
-          dev: `yarn setup && next dev`,
+          dev: `yarn setup && next ${
+            shouldRunTurboDevTest() ? 'dev --turbo' : 'dev'
```

Comment:
This shouldn't be necessary as the `TURBOPACK` env var also switches next dev to turbopack

Reply:
no, actually it makes some difference for some reason at least when I tried locally.

Reply:
hmm... weird... it shouldn't behave differently.

The only inconsistency I found was that: https://github.com/vercel/next.js/pull/55379

Reply:
Don't have good explanation too, but since this is test fixture setup I wanted to make it work first and think about the improvement later as it can wait.